### ISSUE TITLE
Mentionable Text color reset for landscape mode

### ISSUE THREAD

**Issue by zoubarry** [linkedin/Spyglass#113]
Insert an at mention in portrait mode and notice the color is highlighted for mentionables as expected, then rotate the device to landscape -> The mentionables' color reset to black

**Comment by nhibner** [comment]
@zoubarry are you able to reproduce with the demo app? Usually, this just means that your implementation of `Mentionable` needs to implement the `Parcelable` interface to have its view state restored properly on reconfiguration.

**Comment by zoubarry** [comment]
@nhibner yes it is reproducible on demo app and my implementation does implement `Parcelable`

**Comment by nhibner** [comment]
Do you mind providing more details on how to reproduce in the demo app (or better yet, attaching a video)? I've tried to reproduce but have not had any luck (testing on a Pixel 4A 5G with Android 11).

**Comment by zoubarry** [comment]
@nhibner  plz see attached

https://user-images.githubusercontent.com/19559767/105137572-5dd24900-5aa8-11eb-9c43-a1d7dadaf213.mp4

**Comment by nhibner** [comment]
Ah yes, that's the extractUi view displayed by the IME in landscape by default. I'm not aware of any way to customize it, as it is technically a completely separate TextView from the one displayed in the app. It's not clear why the spans are not being persisted to it when copying the text. Honestly, I have not spent much time looking into it, but if there's something we can do to make this work more seamlessly, I'd be open to merging a fix.

**Comment by zoubarry** [comment]
@nhibner ic yea thats probably the default android landscape IME behavior then. Feel free to close
